libyul/AsmAnalysis.cpp

```diff
@@ -257,12 +257,17 @@ vector<YulString> AsmAnalyzer::operator()(FunctionCall const& _funCall)
 	vector<YulString> const* returnTypes = nullptr;
 	vector<bool> const* needsLiteralArguments = nullptr;
 
+	bool runWarnOnInstructions = false;
+	bool reportDeclarationError = false;
+
 	if (BuiltinFunction const* f = m_dialect.builtin(_funCall.functionName.name))
 	{
 		parameterTypes = &f->parameters;
 		returnTypes = &f->returns;
 		if (f->literalArguments)
 			needsLiteralArguments = &f->literalArguments.value();
+
```

Comment:
What do you think about just adding a check here `if (f.isPc) // add warning`? Then you can remove the rest of the new code here ( and also also in `AsmAnalyzer::warnOnInstructions`)

Reply:
the first review however asked me to put the check **into** warnOnInstructions(). It also kinda mekes sense to keep it there since PC is an instruction that we warn (later fail) on.

Reply:
I see. In that case, this is fine. The member `isPc` is not used anywhere. Is it needed?